packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -28,6 +28,7 @@ import LinkViewer from '../url-popover/link-viewer';
 
 const MediaReplaceFlow = ( {
 	mediaURL,
+	mediaId,
```

Comment:
mediaId is passed to MediaUpload as value and here is passed as mediaId. Should we be consistent and update the name here?